core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStoreReaderThread.java

```diff
@@ -171,11 +165,14 @@ public void doWork() {
             log.error("Failed to add record from the Kafka topic" + topic + " the local store");
           }
         }
+
+        if (commitInterval > 0 && System.currentTimeMillis() - lastCommitTime > commitInterval) {
```

Comment:
Looking back on this, I think we want to a) remove commit-related logic entirely, or b) actually design a snapshot mechanism that makes use of commits.

I don't see (b) as happening, or as particularly necessary, so probably we want to remove commit logic entirely.